src/components/Header/Header.test.tsx

```diff
@@ -24,42 +25,20 @@ const mockLogout = jest.fn()
 describe('Header component', () => {
   let user: ReturnType<typeof userEvent.setup>
   beforeEach(() => {
-    renderWithAuthAndApollo(<Header />, { logout: mockLogout }, getThemeMock)
+    renderWithAuthAndApollo(<Header />, { logout: mockLogout }, [
+      getThemeMock,
+      getSiteHeaderMock,
+    ])
     user = userEvent.setup()
   })
-  it('renders the USSF portal header', () => {
+  test('renders the USSF portal header', () => {
     expect(
       screen.getByRole('img', { name: 'United States Space Force Logo' })
     ).toHaveAttribute('alt', 'United States Space Force Logo')
-    expect(screen.getAllByRole('link')).toHaveLength(2)
+    expect(screen.getAllByRole('link')).toHaveLength(1)
   })
-  it('can open the About Us dropdown on click and close on mouse leave', async () => {
```

Comment:
Question: Does making the mock data have an About Us dropdown lets this test pass?

Reply:
Thanks for the explanation!